uplink/storage/segments/store.go

```diff
@@ -71,56 +71,13 @@ func NewSegmentStore(metainfo *metainfo.Client, ec ecclient.Client, rs eestream.
 }
 
 // Put uploads a segment to an erasure code client
-func (s *segmentStore) Put(ctx context.Context, streamID storj.StreamID, data io.Reader, expiration time.Time, segmentInfo func() (int64, storj.SegmentEncryption, error)) (meta Meta, err error) {
+func (s *segmentStore) Put(ctx context.Context, data io.Reader, expiration time.Time, limits []*pb.AddressedOrderLimit, piecePrivateKey storj.PiecePrivateKey) (_ []*pb.SegmentPieceUploadResult, size int64, err error) {
 	defer mon.Task()(&ctx)(&err)
 
-	peekReader := NewPeekThresholdReader(data)
-	remoteSized, err := peekReader.IsLargerThan(s.thresholdSize)
-	if err != nil {
-		return Meta{}, err
-	}
-
-	if !remoteSized {
-		segmentIndex, encryption, err := segmentInfo()
-		if err != nil {
-			return Meta{}, Error.Wrap(err)
-		}
-
-		err = s.metainfo.MakeInlineSegment(ctx, metainfo.MakeInlineSegmentParams{
-			StreamID: streamID,
-			Position: storj.SegmentPosition{
-				Index: int32(segmentIndex),
-			},
-			Encryption:          encryption,
-			EncryptedInlineData: peekReader.thresholdBuf,
-		})
-		if err != nil {
-			return Meta{}, Error.Wrap(err)
-		}
-		return Meta{}, nil
-	}
-
-	segmentIndex, encryption, err := segmentInfo()
-	if err != nil {
-		return Meta{}, Error.Wrap(err)
-	}
-
-	segmentID, limits, piecePrivateKey, err := s.metainfo.BeginSegment(ctx, metainfo.BeginSegmentParams{
-		StreamID:      streamID,
-		MaxOrderLimit: s.maxEncryptedSegmentSize,
```

Comment:
`segmentStore.maxEncryptedSegmentSize` field isn't used anymore, we can remove it.

Reply:
done